codepipeline_deploy.sh

```diff
@@ -12,6 +11,22 @@ if [ -z "${PIPELINE_TIMEOUT}" ]; then
     PIPELINE_TIMEOUT=1800
 fi
 
+# Get the HEAD commit ID for version.json in master branch if exists
```

Comment:
CodeCommit APIs are not the easiest to work with, might've been easier to use `git` directly (clone, add, commit etc). As a bonus with git it would be easier to tweak this entire repo to be useful for the community as a whole, as the clone URL could be passed from .travis.yml and folks could use any repository hosting solution they want, including GitHub.
I'm not suggesting you change it now, just a thought to consider in the future :)

Reply:
I have remove use of codecommit to upload the version.json file. please take a look.